### ISSUE TITLE
Recurrent splenomegaly

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10537]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
I have found the word intermittent as a synonym for recurrent, however, the term Fluctuating splenomegaly does not seem to me to correspond exactly.
I'm not sure it can be added as a synonym, that's why I asked for a new term, but we can discuss it.

PMID: 25666262 The X-linkedinhibitor of apoptosis (XIAP)deficiency, also knownas theX-linkedlymphoproliferative syndrome type 2 (XLP-2), is a rare primary immunodeficiency. XIAP deficiency is characterized by a key triad of clinical manisfestations, which consist of a high susceptibility to develop hemophagocytic lymphohistiocytosis (HLH) frequently triggered by Epstein–Barr virus (EBV) infection, recurrent splenomegaly and inflammatory bowel disease (IBD) with the features of a Crohn’s disease.

PMID: 34222142 Splenomegaly is a classical manifestation of XIAP deficiency, with approximately half of the patients having persistent splenomegaly or experiencing one or more episodes of splenomegaly.

PMID: 21119115 Recurrent splenomegaly often associated with cytopenia and fever was preferentially observed in XLP-2 (XLP-1, 7%; XLP-2, 87%) and probably represents minimal forms of HLH as documented by histopathology. 

**Parent term (use hpo.jax.org/app)**
Splenomegaly HP:0001744

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
X-linked lymphoproliferative disease due to XIAP deficiency (X-linked lymphoproliferative syndrome type 2 or XLP2) ORPHA:538934

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
I would suggest it is better to use the modifer term [Recurrent HP:0031796](https://hpo.jax.org/app/browse/term/HP:0031796).
I am not sure that Fluctuating splenomegaly is the greatest term, and will replace those annotations using the modifer [Fluctuating HP:0031914](https://hpo.jax.org/app/browse/term/HP:0031914)

**Comment by carolina-f** [comment]
Thank you, but at the moment I do not have access to another branch other than Phenotypic abnormality to phenotype diseases in Orphanet

**Comment by pnrobinson** [comment]
@carolina-f 
I think it would be great if Orphanet and HPO could collaborate more closely on annotations and use the same annotation model, we could set up a zoom call to discuss. But it would be good to use the Recurrent term here.
